pkg/smokescreen/acl_loader_v1.go

```diff
@@ -62,16 +62,16 @@ func (ew *EgressAclConfig) Decide(fromService string, toHost string) (EgressAclD
 	case ConfigEnforcementPolicyEnforce:
 		action = EgressAclDecisionDeny
 	case ConfigEnforcementPolicyOpen:
-		return EgressAclDecisionAllow, defaultRuleUsed, nil
+		return EgressAclDecisionAllow, "", defaultRuleUsed, nil
 	default:
-		return 0, defaultRuleUsed, fmt.Errorf("unexpected policy value for (%s -> %s): %d", fromService, toHost, rule.Policy)
+		return 0, "", defaultRuleUsed, fmt.Errorf("unexpected policy value for (%s -> %s): %d", fromService, toHost, rule.Policy)
 	}
 
 	// use the decision from rule.Policy
-	return action, defaultRuleUsed, nil
+	return action, "", defaultRuleUsed, nil
```

Comment:
On line 62 we can also assign a reason, and return it alongside the action that was selected here. That way when we hit this case, we can log which policy was selected, and that the domain fell through to the policy action.